src/consensus/tx_check.cpp

```diff
@@ -7,28 +7,28 @@
 #include <primitives/transaction.h>
 #include <consensus/validation.h>
 
-bool CheckTransaction(const CTransaction& tx, CValidationState& state)
+bool CheckTransaction(const CTransaction& tx, TxValidationState &state)
```

Comment:
nit: the `&` was moved from the type to the variable name. For consistency I would keep it at end of the type. That's also how it is still declared in the header file.